WTWSMS/common/event_modifiers/WtWSMS_event_modifiers.txt

```diff
@@ -915,6 +878,101 @@ secret_taoism_community = {
 	}
 }
 
+<<<<<<< HEAD
```

Comment:
You are wrong. **_Yesterday_**, I had to synchronize with your new updates to the Master line of the Mod. When Tanner Committed the code, we had both already tested it in game. This mistake is _brand new_.

You are absolutely wrong.